templates/main_template.yml

```diff
@@ -11,6 +11,7 @@ jobs:
       fail-fast: false
     permissions:
       contents: write
+    if: "!contains(github.event.head_commit.message, '[ci]')"
```

Comment:
Nice to have. Sometimes you'll be changing things that have nothing to do with the plugin itself ( e.g. readme file ). No reason to run the workflow in those cases, so we can do it by adding [ci] inside the commit name.